worker/games.py

```diff
@@ -1567,8 +1567,12 @@ def make_player(arg):
                 os.path.join(testing_dir, fastchess),
                 "-recover",
                 "-repeat",
+                "-ratinginterval",
+                "0",
+                "-autosaveinterval",
+                "0",
                 "-games",
-                str(int(games_to_play)),
+                str(int(games_to_play) // 2),
```

Comment:
can games to play be odd ?

Reply:
> can games to play be odd ?

theres an assert that prevents it, so it will always be even https://github.com/vondele/fishtest/blob/c271145467afa568f234d6a69f79e5db46ff6680/worker/games.py#L1538